modules/localgov_forms_example_liberty_create_integration/src/PrepareFileTokens.php

```diff
@@ -0,0 +1,155 @@
+<?php
```

Comment:
Should this file be in the module root instead of localgov_forms_examples?

Reply:
Not sure Andy.  The token from this file is not needed anywhere other than Liberty Create API integration.  So I thought it better to just leave it in the demo submodule.